CMakeLists.txt

```diff
@@ -426,6 +426,26 @@ if(NOT WIN32)
 endif()
 
 find_package(Readline REQUIRED)
+find_package(Augeas REQUIRED)
+
+if(LINUX)
+  find_package(LibSELinux REQUIRED)
+endif()
+
+if(APPLE)
+  if(NOT DEFINED ENV{BUILD_LINK_SHARED})
```

Comment:
Oh now I remember. libxml2 places it's headers in /usr/include/libxml2/libxml and people include it with `#include <libxml/header.h>`.
It's the same on Linux and OSX.